src/scvi/external/methylvi/_utils.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from typing import Optional, Union
+
+import numpy as np
+from mudata import MuData
+
+from scvi.data import AnnDataManager
+
+logger = logging.getLogger(__name__)
+
+
+def scmc_raw_counts_properties(
+    adata_manager: AnnDataManager,
+    idx1: Union[list[int], np.ndarray],
+    idx2: Union[list[int], np.ndarray],
+    var_idx: Optional[Union[list[int], np.ndarray]] = None,
+    modality: str = None,
+) -> dict[str, np.ndarray]:
+    """Computes and returns some statistics on the raw counts of two sub-populations.
+
+    Parameters
+    ----------
+    adata_manager
+        :class:`~scvi.data.AnnDataManager` object setup with :class:`~scvi.model.SCVI`.
+    idx1
+        subset of indices describing the first population.
+    idx2
+        subset of indices describing the second population.
+    mc_key
+        denotes layer where methylated cytosine counts are stored in AnnData
+    cov_key
+        denotes layer where total cytosine counts are stored in AnnData
+    var_idx
+        subset of variables to extract properties from. if None, all variables are used.
+
+    Returns
+    -------
+    type
+        Dict of ``np.ndarray`` containing, by pair (one for each sub-population).
+    """
+    adata = adata_manager.adata
+
+    if isinstance(adata, MuData):
```

Comment:
Do we still need this one, if we require MuData input?

Reply:
Thanks for the catch. Fixed!